### PR TITLE
AIR-1898 (Change link to Forum from Sahara to the new Forum UI)

### PR DESCRIPTION
**Pull request by jialin-he**

### CODE REVIEW

File: src/app/login/login.component.pug [hunk 1]
```diff
@@ -48,7 +48,7 @@
           h1#saharaLoginHeading {{ copyBase + 'LOGIN.SAHARA_LOGIN.HEADING' | translate }}
         .card-body
           .form-group
-            | #[a#linkLoginSaharaSS.link(tabindex="0", href="http://catalog.sharedshelf.artstor.org/login.html", target="_blank", data-automation-component="ss-login-link") Log in] to Forum
+            | #[a#linkLoginSaharaSS.link(tabindex="0", [attr.href]="prod ? ('FORUM.PROD' | translate) :('FORUM.TEST' | translate)", target="_blank", data-automation-component="ss-login-link") Log in] to JSTOR Forum
```
Comment (codiak) [inline src/app/login/login.component.pug]:
```suggestion
            | #[a#linkLoginSaharaSS.link(tabindex="0", [attr.href]="(prod ? 'FORUM.PROD' : 'FORUM.TEST') | translate", target="_blank", data-automation-component="ss-login-link") Log in] to JSTOR Forum
```

Comment (codiak) [reply src/app/login/login.component.pug]: Could be a tad more succinct. What do you think?

**Comment by bFraley** [comment]
@codiak @rehanabbasi - I included 1936 in this PR, so please don't merge until we know which footer link to use for 1936

**Comment by bFraley** [comment]
@codiak @rehanabbasi Disregard the previous comment, I'm not including 1936 logic in this PR

**Pull request merged**